app/controllers/teams_controller.rb

```diff
@@ -137,4 +138,8 @@ def add_team_for(user)
       flash[:notice] = t('teams.team_was_successfully_updated')
     end
   end
+
+  def check_recaptcha
+    (show_recaptcha?)? verify_recaptcha : true
```

Comment:
Is this a ternary? Maybe you can come with something like:

```
show_recaptcha? && verify_recaptcha
```

I'm quite sure this `true` is avoidable 🤔

Reply:
Besides this expression `(show_recaptcha?)?` is not usual 🤔

Reply:
Omit parentheses for ternary conditions or use somathing like @dmitryrck said